scripts/change_base_branch.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
```

Comment:
maybe `set -e -o pipefail` ?

Reply:
Sure! There is a feature in comments "add suggestion" so I will need just to click a button "apply" :)